share/completions/emerge.fish

```diff
@@ -1,124 +1,176 @@
+complete -c emerge      -l metadata
+complete -c emerge -s P -l prune
+complete -c emerge      -l regen
+complete -c emerge -s r -l resume
+complete -c emerge -s s -l search
+complete -c emerge -s S -l searchdesc
+complete -c emerge      -l sync
+complete -c emerge -s C -l unmerge
+complete -c emerge -s V -l version
 
 #########################
-# Options
-complete -c emerge -l alphabetical -d "Sort flag lists alphabetically"
-complete -c emerge -s a -l ask -d "Prompt the user before peforming the merge"
-complete -c emerge -s b -l buildpkg -d "Build a binary package additionally"
-complete -c emerge -s B -l buildpkgonly -d "Only build a binary package"
-complete -c emerge -s l -l changelog -d "Show changelog of package. Use with --pretend"
-complete -c emerge -l color -d "Toggle colorized output" -xa \
-'y\t"'(_ "Use colors in output")'" '\
-'n\t"'(_ "Don't use colors in output")'"'
-
-complete -c emerge -l colums -d "Align output. Use with --pretend"
-complete -c emerge -s d -l debug -d "Run in debug mode"
-complete -c emerge -s D -l deep -d "Consider the whole dependency tree"
-complete -c emerge -s e -l emptytree -d "Reinstall all world packages"
-complete -c emerge -s f -l fetchonly -d "Only download the packages but don't install them"
-complete -c emerge -s F -l fetch-all-uri -d "Same as --fetchonly and grab all potential files"
-complete -c emerge -s g -l getbinpkg -d "Download infos from each binary package. Implies -k"
-complete -c emerge -s G -l getbinpkgonly -d "As -g but don't use local infos"
-complete -c emerge -l ignore-default-opts -d "Ignore EMERGE_DEFAULT_OPTS"
-complete -c emerge -s N -l newuse -d "Include installed packages with changed USE flags"
-complete -c emerge -l noconfmem -d "Disregard merge records"
-complete -c emerge -s O -l nodeps -d "Don't merge dependencies"
-complete -c emerge -s n -l noreplace -d "Skip already installed packages"
-complete -c emerge -l nospinner -d "Disable the spinner"
-complete -c emerge -s 1 -l oneshot -d "Don't add packages to world"
-complete -c emerge -s o -l onlydeps -d "Only merge dependencies"
-complete -c emerge -s p -l pretend -d "Display what would be done without doing it"
-complete -c emerge -s q -l quit -d "Use a condensed output"
-complete -c emerge -l skipfirst -d "Remove the first package in the resume list. Use with --resume"
-complete -c emerge -s t -l tree -d "Show the dependency tree"
-complete -c emerge -s k -l usepkg -d "Use binary package if available"
-complete -c emerge -s K -l usepkgonly -d "Only use binary packages"
-complete -c emerge -s v -l verbose -d "Run  in  verbose  mode"
-complete -c emerge -l with-bdeps -d "Toggle build time dependencies" -xa \
-'y\t"'(_ "Pull in build time dependencies")'" '\
-'n\t"'(_ "Don't pull in build time dependencies")'"'
-
-# END Options
+# Options               #
 #########################
-
+complete -c emerge -s A -l alert        -d "add a terminal bell character ('\a') to all interactive prompts"
```

Comment:
Does the "interactive" add anything here? Or would just `Add a bell ('\a') to all prompts` suffice?

Reply:
As far as I can tell, `-A` without `-a` (or any other flag that causes interaction) has no effect.